src/storage/src/source/kafka.rs

```diff
@@ -1102,7 +1032,7 @@ impl KafkaSourceReader {
                 source_id = self.id.to_string(),
                 worker_id = self.worker_id,
                 num_workers = self.worker_count,
-                "kafka message before expected offset, skipping: \
+                "kafka message before expected offset: \
```

Comment:
the comment above `        // If we see an "old" offset, we ast-forward the consumer and skip that message` needs updating -- I couldn't comment on that line directly